crates/ludtwig-parser/src/grammar/twig/tags.rs

```diff
@@ -5167,4 +5205,32 @@ mod tests {
                 error at 9..11: expected twig expression as cache key but found %}"#]],
         );
     }
-}
+
+  #[test]
+  fn parse_twig_trans() {
+      check_parse(
+          "{% trans %} hello world {% endtrans %}",
+          expect![[r#"
+              ROOT@0..38
+                TWIG_TRANS@0..38
```

Comment:
So this is basically your `TWIG_TRANS` element and if I look at the provided twig code I would expect it to contain:
- `TWIG_TRANS_STARTING_BLOCK` => `{% trans %}`
- `BODY` => ` hello world`
- `TWIG_TRANS_ENDING_BLOCK` => ` {% endtrans %}`

Right now it contains `TWIG_TRANS ` two times (the inner is the starting block) and is missing the `BODY` wrapper around all child nodes.